_posts/en/newsletters/2021-04-14-newsletter.md

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Bitcoin Optech Newsletter #144'
+permalink: /en/newsletters/2021/04/14/
+name: 2021-04-14-newsletter
+slug: 2021-04-14-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Taproot activation discussion:** since our last update on discussion
+  about [activation methods][topic soft fork activation] for the
+  [taproot][topic taproot] soft fork in [Newsletter #139][news139
+  activation], the
+  Speedy Trial proposal became the focus of attention among those
+  interested in activation.  PRs were opened for two variants of it:
+  [PR#21377][Bitcoin Core #21377], using a variation on [BIP9][], and
+  [PR#21392][Bitcoin Core #21392], using a modification that became part
+  of [BIP8][].  The main technical difference between these PRs is how
+  their start and stop points were specified.  PR#21377 uses Median Time
+  Past (MTP)---the median time declared in the headers of the current
+  and previous 11 blocks.  PR#21392 uses the *height* of the current
```

Comment:
```suggestion
  and previous 10 blocks.  PR#21392 uses the *height* of the current
```

The median time past is the median time of "the last 11 blocks", or "this block and its 10 predecessor blocks".